tests/test_gossip.py

```diff
@@ -1878,13 +1910,27 @@ def channel_fees(node):
             '0102c479b7684b9db496b844f6925f4ffd8a27c5840a020d1b537623c1545dcd8e195776381bbf51213e541a853a4a49a0faf84316e7ccca5e7074901a96bbabe04e06226e46111a0b59caaf126043eb5bbf28c34f3a5e332a1fc7b2b73cf188910f00006700000100015d77400201010006000000000000000000000014000003eb000000003b023380',
             # timestamp=1568096259, fee_proportional_millionths=1004
             '01024b866012d995d3d7aec7b7218a283de2d03492dbfa21e71dd546ec2e36c3d4200453420aa02f476f99c73fe1e223ea192f5fa544b70a8319f2a216f1513d503d06226e46111a0b59caaf126043eb5bbf28c34f3a5e332a1fc7b2b73cf188910f00006700000100015d77400301010006000000000000000000000014000003ec000000003b023380',
-            # update 5 marks you as a nasty spammer!
+            # update 5 marks you as a nasty spammer, but we listen to you anyway now! fee_proportional_millionths=1005
             '01025b5b5a0daed874ab02bd3356d38190ff46bbaf5f10db5067da70f3ca203480ca78059e6621c6143f3da4e454d0adda6d01a9980ed48e71ccd0c613af73570a7106226e46111a0b59caaf126043eb5bbf28c34f3a5e332a1fc7b2b73cf188910f00006700000100015d77400401010006000000000000000000000014000003ed000000003b023380'
         ],
         timeout=TIMEOUT
     )
-
-    wait_for(lambda: channel_fees(l3) == [1004])
+    # Rate limited channel_update okay to use in routing graph.
+    wait_for(lambda: channel_fees(l3) == [1005])
+    # but should be flagged so we don't propagate to the network.
+    assert(l3.daemon.is_in_log("Spammy update for 103x1x1/1 flagged"))
+
+    messages = serialize_gossipwith_output(l3)
+    for m in messages:
+        print(m)
+    decoded = subprocess.run(['devtools/decodemsg', messages[-1]],
+                             check=True,
+                             timeout=TIMEOUT,
+                             stdout=subprocess.PIPE).stdout.decode('utf8')
+    print("last gossipwith output:")
+    print(decoded)
```

Comment:
this can be removed? not sure that without enabling the console in the pytest this information is displayed, maybe I'm wrong?

Reply:
Oops, I have some other cleanup in this test as well.  Rushed this one out hoping to get some feedback on optimal broadcast behavior.  I'm leaning toward sending out old gossip instead of skipping entirely.  It's going to complicate indexing and compacting the gossip_store, so I was hoping to determine that's really the best course of action.